Sources/SwipeTransition/SwipeBack/SwipeBackController.swift

```diff
@@ -58,14 +58,26 @@ public final class SwipeBackController: NSObject {
         context.navigationControllerDelegateProxy = NavigationControllerDelegateProxy(delegates: [self] + (delegate.map { [$0] } ?? []) )
     }
 
-    public func observePageViewController(_ pageViewController: UIPageViewController, isFirstPage: @escaping () -> Bool) {
+    public func observePageViewController(_ pageViewController: UIViewController, shouldBeginSwipe: @escaping (UIGestureRecognizer?) -> Bool) {
```

Comment:
I want to change the name to "observe" to unify the naming convention.
```suggestion
    public func observe(viewController: UIViewController, shouldBeginSwipe: @escaping (UIGestureRecognizer) -> Bool) {
```